_rules/descriptive-label-cc0f0a.md

```diff
@@ -121,15 +124,15 @@ Label that is coded with the `p` element and associated by the aria-labelledby a
 <input aria-labelledby="label_fname" type="text" name="fname" />
 ```
 
-#### Failed example 3
+#### Failed Example 3
 
 Implicit label that is coded with the `label` element and does not describe the purpose of the associated element.
 
 ```html
 <label>Menu<input id="fname" type="text" name="fname"/></label>
 ```
 
-#### Passed example 4
+#### Passed Example 4
```

Comment:
Now we are at it, should we fix these ones up so that they say "Failed Example X" when they are part of the "Failed" section :)